chart/templates/ws-proxy-configmap.yaml

```diff
@@ -60,6 +60,15 @@ data:
                 "location": "/app/public"
             }
         },
+        {{ if (and $comp.wsManagerProxy $comp.wsManagerProxy.enabled) }}
```

Comment:
bonus feature: the helm chart could fail if someone were to expose their ws-manager but had no auth/TLS configured.

Reply:
Good idea. But we don't have TLS config in ws-manager template yet:

https://github.com/gitpod-io/gitpod/blob/555ca137b0f68d02caa1b18248b46f34619b5fd5/chart/templates/ws-manager-configmap.yaml#L95-L111

Already working on a follow-up PR to add this.